budgetportal/datasets.py

```diff
@@ -369,3 +369,21 @@ def get_consolidated_expenditure_budget_dataset(financial_year):
         return Dataset.from_package(package)
     else:
         return None
+
+
+def get_budgeted_and_actual_comparison_dataset(financial_year):
```

Comment:
this is not  the budget and actual dataset, it's the in-year spending dataset.

```suggestion
def get_in_year_spending_dataset(financial_year):
```

Reply:
done